app/assets/javascripts/views/story_view.js

```diff
@@ -647,7 +629,7 @@ module.exports = FormView.extend({
           name='attachments'
           isReadonly={this.isReadonly()}
           filesModel={this.model.get('documents')}
-          options={this.attachmentOptions}
+          //options={this.attachmentOptions}
```

Comment:
?

Reply:
this comment, I put to show to @talyssonoc  because I dont know if this part is essencial or can remove but I think it will be remove.